src/components/SpruceForm/index.tsx

```diff
@@ -13,31 +14,37 @@ import widgets from "./Widgets";
 
 export const SpruceForm: React.VFC<SpruceFormProps> = ({
   customFormatFields,
+  customValidate,
   disabled,
   fields,
   formData,
   onChange,
   schema,
   tagName,
   uiSchema,
-  validate,
 }) => (
   <Form
-    fields={{ ...baseFields, ...fields }}
+    fields={fields}
     schema={schema}
     onChange={onChange}
     widgets={widgets}
     uiSchema={uiSchema}
     formData={formData}
     tagName={tagName}
-    ArrayFieldTemplate={ArrayFieldTemplate}
-    FieldTemplate={DefaultFieldTemplate}
-    ObjectFieldTemplate={ObjectFieldTemplate}
+    templates={{
+      ArrayFieldTemplate,
+      FieldTemplate: DefaultFieldTemplate,
+      ObjectFieldTemplate,
+      DescriptionFieldTemplate: baseFields.DescriptionField,
+      TitleFieldTemplate: baseFields.TitleField,
+    }}
```

Comment:
Extracting these into an object outside of this component will avoid redeclaring every render.

Reply:
Also you can use the TemplatesType